extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/KeycloakDevConsoleProcessor.java

```diff
@@ -46,6 +46,8 @@ public void setConfigProperties(BuildProducer<DevConsoleTemplateInfoBuildItem> c
             console.produce(new DevConsoleTemplateInfoBuildItem("oidcGrantType", config.devservices.grant.type.getGrantType()));
             console.produce(new DevConsoleTemplateInfoBuildItem("swaggerIsAvailable",
                     capabilities.isPresent(Capability.SMALLRYE_OPENAPI)));
+            console.produce(new DevConsoleTemplateInfoBuildItem("graphqlIsAvailable",
```

Comment:
Would it make sense not to register this item at all if it is not available (and same for the earlier swagger UI item) ? Then in `provider.html` it would be a `graphqlIsAvailable??`

Reply:
I guess that would work too, I kept it the same as the swagger ui code. I have no real preference, let me know if you want this changed

Reply:
Ok let me change that for both. Please hold